app/views/account/dashboard/index.html.haml

```diff
@@ -155,7 +155,8 @@
         .box-content
           .statistics-box.with-icon
             %i.ico.ti-user.text-primary
-            %h2.counter.text-inverse Hello, user
+            %h2.counter.text-inverse Hello
+            = current_user.first_name if user_signed_in?
```

Comment:
remove this

 if user_signed_in?

USER must be always logged in to see the dashboard